ts/features/euCovidCert/__e2e__/euCovidCertExpired.e2e.ts

```diff
@@ -7,7 +7,7 @@ const euCovidCertExpiredSubject = "🏥 EUCovidCert - expired";
 const euCovidCertExpiredTitle = "Expired Certificate title";
 const euCovidCertExpiredSubTitle = "Expired Certificate sub title";
 
-const messageListTestId = "messageList";
+const messageListTestId = "MessageList_inbox";
```

Comment:
Why are we testing pagination under `euCovidCertExpired.e2e.ts` ? It is already covered in `messages.e2e.ts` 🤔

Reply:
Well e2e tests are also made for this. They are focused on end-user experience, so it's normal to touch different pieces on the app on a single use-case. However I found out that by removing the "load all messages" step, all the other tests pass regularly. But the "scroll until you find this message" is something you cannot avoid.